deploy/modules/aws/tidb-cluster/values/default.yaml

```diff
@@ -14,6 +14,17 @@ tidb:
       service.beta.kubernetes.io/aws-load-balancer-type: nlb
       service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
   separateSlowLog: true
+  config: |
+    [log]
+    level = "info"
+    [performance]
+    tcp-keep-alive = true
+  podSecurityContext:
+    sysctls:
+    - name: net.ipv4.tcp_keepalive_time
+      value: "300"
+    - name: net.ipv4.tcp_keepalive_intvl
```

Comment:
Should we put these as default configs in values.yaml of tidb-cluster chart?

Reply:
No, these configurations are specific to AWS.

For `net.core.somaxconn`, I think it's another problem and can be addressed in an separate issue.

Reply:
BTW, what's the proper value of `net.core.somaxconn`?

Reply:
32768

ref: https://pingcap.com/docs/v3.0/tidb-in-kubernetes/deploy/prerequisites/

Reply:
The `net.core.somaxconn` is a general issue, so I think we can set this in the tidb-cluster chart values.yaml

Reply:
not possible, it's marked unsafe because of kernel memory accounting issue, so it must be whitelisted via kubelet flag, otherwise the pod will fail to start

Reply:
oh, I misunderstood here, it's ok to add in deploy/aws values.yaml file.

Reply:
Most of the kernel parameters in the prerequisites document are namespaced, seems like we should configure the safe part for users by default and add document about how to configure these parameters via pod security context. Tracked in: #924